src/writeData/subscriptions/components/BrokerFormContent.tsx

```diff
@@ -274,10 +282,15 @@ const BrokerFormContent: FC<Props> = ({
             weight={FontWeight.Regular}
             className={`${className}-broker-form__example-text`}
           >
-            {/* TODO: update `false` after cert support */}
-            {getSchemaFromProtocol(formContent.protocol, false)}
+            {getSchemaFromProtocol(
+              formContent.protocol,
+              isFlagEnabled('subscriptionsCertificateSupport') &&
+                formContent.authType === BrokerAuthTypes.Certificate
+            )}
             {formContent.brokerHost ? formContent.brokerHost : '0.0.0.0'}:
-            {formContent.brokerPort ? formContent.brokerPort : '1883'}
+            {formContent.brokerPort
+              ? formContent.brokerPort
+              : `${DEFAULT_PORT}`}
```

Comment:
Do we need this ternary now that brokerPort has a default value in the context?

Reply:
yes, because if you delete the port then you will see something like `0.0.0.0:` which is supposed to show what the url should look like.

Reply:
Ship it

Reply:
Right but that placeholder would be accurate, right? Because if you delete the port, the address is invalid. So we should show the user the invalid address so they understand what needs to be fixed. Or am I missing something?

Reply:
i agree with gene here if we have a default port we no longer need a ternary